server/testkit/tests/proto_v1_test.rs

```diff
@@ -532,6 +532,12 @@ async fn test_server_rest_posix_lifecycle(rsclient: KanidmClient) {
         .idm_group_unix_extend("posix_group", None)
         .await
         .unwrap();
+    // here we check that we can successfully change the gid without breaking anything
```

Comment:
Change another attribute and check the gid stays the same ;)

Reply:
what other attributes can I change besides the gid? Because in `GroupUnixExtend` I just see `pub gidnumber: Option<u32>,`

Reply:
🤔 good point. But if the person runs the command with a "None" in the gid it should be idempotent and not change the gid.

Reply:
agreed, so I'll ensure that won't happen